security_monitoring/README.md

```diff
@@ -0,0 +1,50 @@
+# Agent Check: security_monitoring
+
+## Overview
+
+This check in theory monitors [security_monitoring][1] through the Datadog Agent. However, the integration currently doesn't provide any checks, but only saved views.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+The security_monitoring check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `security_monitoring.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your security_monitoring performance data. See the [sample security_monitoring.d/conf.yaml][3] for all available configuration options.
+
+2. [Restart the Agent][4].
+
+### Validation
+
+[Run the Agent's status subcommand][5] and look for `security_monitoring` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][6] for a list of metrics provided by this check.
+
+### Service Checks
+
+security_monitoring does not include any service checks.
+
+### Events
+
+security_monitoring does not include any events.
```

Comment:
I would remove all of this since it doesn't apply.

Reply:
The CI requires some of these subsections. I think the readme otherwise makes it clear that there are no checks provided.

Reply:
Correct, some of the docs build can break if sections are missing from the integration readme.